viz_scripts/energy_calculations.ipynb

```diff
@@ -41,18 +41,13 @@
    "source": [
     "from collections import defaultdict\n",
     "import datetime\n",
+    "import sys\n",
+    "\n",
     "import matplotlib.pyplot as plt\n",
```

Comment:
Do you even need `import sys`

Reply:
No, also removed some other imports that are no longer needed